lib/uniswap/swap_test.go

```diff
@@ -77,6 +81,11 @@ func TestSwapToUSDC(t *testing.T) {
 			asset:    tokens.WSTETH,
 			amountIn: bi.Ether(1),
 		},
+		{
+			name:     "USDT to USDC",
+			asset:    tokens.USDT,
+			amountIn: bi.Ether(1),
+		},
```

Comment:
This isn't working yet. Need to get the slot index FundERC20 option from other PR in and use that